releasenotes/notes/collector-104-8697c1d5cc4b1745.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Change default bind address in OTLP ingest from `0.0.0.0` to `localhost`. Please refer to the following blog post: https://opentelemetry.io/blog/2024/hardening-the-collector-one for additional information around this change.
```

Comment:
The release note has nothing to do with the title of the PR, is that intended?

Reply:
The bump to v0.104.0 brings in the change mentioned in the changelog, see: https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.104.0.

Reply:
Nit: Maybe mention the upgrade to `0.104.0` of the collector